Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -1447,8 +1548,11 @@ Tileset::TraversalDetails Tileset::_visitTile(
 
   // At least one descendant tile was added to the render list.
   // The traversalDetails tell us what happened while visiting the children.
-  if (!traversalDetails.allAreRenderable &&
-      !traversalDetails.anyWereRenderedLastFrame) {
+  if ((!traversalDetails.allAreRenderable &&
+       !traversalDetails.anyWereRenderedLastFrame) ||
+      (lastFrameSelectionResult == TileSelectionState::Result::Rendered &&
+       tile.getContent() &&
+       tile.getContent()->lodTransitionFadePercentage < 1.0f)) {
```

Comment:
@kring This is another part of the code worth reviewing in particular: the changes to the kick logic. This allows us to keep descendants off the render list if the tile is still fading in. The descendants may still continue to load though.